WORKSPACE

```diff
@@ -50,33 +50,26 @@ load("@io_bazel_rules_go//go:deps.bzl", "go_download_sdk", "go_register_toolchai
 
 go_rules_dependencies()
 
-go_download_sdk(
-    name = "go_sdk_linux",
-    goarch = "amd64",
-    goos = "linux",
-    version = "1.20.3",  # Keep in sync with .github/workflows/checkstyle.yaml
-)
-
-go_download_sdk(
-    name = "go_sdk_linux_arm64",
-    goarch = "arm64",
-    goos = "linux",
-    version = "1.20.3",
-)
-
-go_download_sdk(
-    name = "go_sdk_darwin",
-    goarch = "amd64",
-    goos = "darwin",
-    version = "1.20.3",
-)
-
-go_download_sdk(
-    name = "go_sdk_darwin_arm64",
-    goarch = "arm64",
-    goos = "darwin",
-    version = "1.20.3",
-)
+# Register multiple Go SDKs so that we can perform cross-compilation remotely.
+# i.e. We might want to trigger a Linux AMD64 Go build remotely from a MacOS ARM64 laptop.
+#
+# Reference: https://github.com/bazelbuild/rules_go/issues/3540.
+[
+    go_download_sdk(
+        name = "go_sdk_{}_{}".format(goos, goarch),
+        goarch = goarch,
+        goos = goos,
+        version = "1.20.4",  # Keep in sync with .github/workflows/checkstyle.yaml
+    )
+    for goos in [
+        "linux",
+        "darwin",
+    ]
+    for goarch in [
+        "amd64",
+        "arm64",
+    ]
+]
```

Comment:
nit: could we just leave these written out?

Reply:
Fixed.